### PR TITLE
Ce/dashboard models

### PR DESCRIPTION
**Pull request by calellowitz**
@emord 
Still working on getting these to be django managed (after removing `managed=False`, `makemigrations` does not seem to pick up changes), but was wondering what you thought of this overall strategy

### CODE REVIEW

**Review by emord — COMMENTED** [review]
Looks good. thanks for doing this.

> Still working on getting these to be django managed (after removing managed=False, makemigrations does not seem to pick up changes), but was wondering what you thought of this overall strategy

I think this is fine if you want to punt it for now. We can continue to make column changes in custom SQL and mirror those changes to the django models. I think this is most useful for having people jump in to change/investigate code. It's not ideal, but it's better than what we have now.

To do this we may need to just wipe all the migrations away and start a new `initial = True` migration that won't create tables if they already exist. I think it makes sense to do that after we investigate what data is no longer used and remove it so we can start with a clean slate.

File: custom/icds_reports/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+from custom.icds_reports.models.util import (
+    UcrTableNameMapping,
+    AggregateSQLProfile
+)
+from custom.icds_reports.models.helper import (
+    ChildHealthCategories,
+    CcsRecordCategories,
+    ThrCategories,
+    IcdsMonths,
+    IndiaGeoData
```
[the remark this answers is not in the document]
Comment (calellowitz) [reply custom/icds_reports/models/__init__.py]: Are you referring to all the tables in `helper.py`

[the remark this answers is not in the document]
Comment (emord) [reply custom/icds_reports/models/__init__.py]: `IcdsMonths` may still be used, I think that the others are not used

**Review by emord — APPROVED** [review]
oh wow. when you said you figured it out, i didn't realize you meant getting this fully in django with future migrations not needing custom sql. Nice.

Looks good from my end when the conflicts are resolved

**Review by emord — APPROVED** [review]
This had a weird error on the build. kicked it so hopefully it was just passing

**Review by emord — COMMENTED** [review]
I think that there should be a migration made right? Even when they're unmanaged, I think there is some state that is tracked

**Comment by emord** [comment]
Looks like 3 tests failed on `curl: (6) Could not resolve host: app.datadoghq.com` so should be able to ignore test failures

**Comment by snopoke** [comment]
:clap: :clap: :clap:

**Pull request merged**